web/App.tsx

```diff
@@ -130,6 +143,7 @@ export default class App extends Component<void, IState> {
                     supportedServices={this.state.supportedServices}
                     serviceScope={this.state.serviceScope}
                     embedType={this.state.embedType}
+                    darkMode={darkMode}
```

Comment:
Passing darkmode as argument like this - while working - seems very explicit and noisy.

An alternative could be to inline SVGs into the DOM which would then allow the compound CSS to be applied to them. I would need to investigate how to do that. (We might not even want compound for this but our own media queries instead to us e.g. the official original GitHub colour, and then media queries are also not MSC2873 which we actually want etc etc).